scripts/s3manage.py

```diff
@@ -0,0 +1,415 @@
+#!/usr/bin/env python
```

Comment:
We want to use the virtualenv python for that so this is not needed as long as s3manage is executable